requirements/base.txt

```diff
@@ -36,7 +36,7 @@ django-autoslug==1.8.0
 pytz==2012h
 
 # background process management
-# django-celery==3.1.16
+django-celery==3.1.16
```

Comment:
We're running the celery package straight now, no need for django-celery, this shouldn't be uncommented.